src/components/SiteFooter/index.js

```diff
@@ -30,7 +30,15 @@ const SiteFooter = () => {
         </Link>
         {/* eslint-disable-next-line react/jsx-curly-brace-presence */}
         {` · `}
-        <Link to="/privacy">Privacy policy</Link>
+        <Link to="/privacy">Privacy</Link>
+        {/* eslint-disable-next-line react/jsx-curly-brace-presence */}
+        {` · `}
+        <Link to="/credits">Credits</Link>
+        {/* eslint-disable-next-line react/jsx-curly-brace-presence */}
+        {` · `}
+        <Link as="a" href="https://github.com/octopusthink/getmicdrop.com">
```

Comment:
I think we should remove the GitHub link, as it implies that the source for the actual app is available on GitHub, which isn't the case and makes this seem a bit misleading 😄

Reply:
I already fixed this last night! Just spaced and forgot to push.